framework/src/main/java/org/checkerframework/framework/util/FlowExpressionParseUtil.java

```diff
@@ -167,12 +220,26 @@ private static boolean isMemberSelect(String s, FlowExpressionContext context) {
      * @param s expression string
      * @return pair of object and field
      */
-    private static Pair<String, String> parseMemberSelect(String s) {
-        Pair<Pair<String, String>, String> method = parseMethodCall(s);
-        if (method != null && method.second.startsWith(".")) {
-            return Pair.of(
-                    method.first.first + "(" + method.first.second + ")",
-                    method.second.substring(1));
+    private static Pair<String, String> parseMemberSelect(String s, FlowExpressionContext context) {
+
+        // replace occurence of formal parameters, such as "#2", with corresponding arguments
+        String sCopy = s;
```

Comment:
`sCopy` is a poor variable name.  It does not indicate what the variable represents or how it is used or how it differs from s.  Even a name containing `replaced` would be better, or one containing `withArgs`.